raysort/main_push.py

```diff
@@ -174,12 +192,16 @@ def _finish_merge_results(self) -> list[list[ray.ObjectRef]]:
         self._merge_tasks = {}
         self._current_merger = None
         ret = self._merge_results
+        ret2 = self._merge_datachunk_sizes
         self._merge_results = []
```

Comment:
should we reset `self._merge_datachunk_sizes` too?

This might be cleaner:

```
ret = (self._merge_results, self._merge_datachunk_sizes)
... = []
return ret
```

Reply:
fixed